From 4cd9ba2cea5b54d472bd00f41ee0d38ef7497521 Mon Sep 17 00:00:00 2001
From: Jorge Luis Martinez Gomez <jol@jol.dev>
Date: Tue, 17 Sep 2019 23:23:58 -0700
Subject: [PATCH] remove redundant escapes causing gawk warnings

---
 txt2man | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/txt2man b/txt2man
index 64d0a4c..abc1c43 100755
--- a/txt2man
+++ b/txt2man
@@ -242,7 +242,7 @@ BEGIN {
 		pnzls = ls
 	match($0, /[^ ]/)
 	ls = RSTART
-	if (pls == 0 && pnzls > 0 && ls > pnzls && $1 !~ /^[0-9\-\*\o]\.*$/) {
+	if (pls == 0 && pnzls > 0 && ls > pnzls && $1 !~ /^[0-9\-\*o]\.*$/) {
 		# example display block
 		if (prevblankline == 1) {
 			print ".PP"
@@ -300,7 +300,7 @@ section == "SYNOPSIS" {
 		} else if ($1 == "#define")
 			subwords["\\<" $2 "\\>"] = "\\fI" $2 "\\fP"
 		for (i = 1; i <= NF; i++) {
-			if ($i ~ /[\,\)]\;*$/) {
+			if ($i ~ /[,\)];*$/) {
 				a = $i
 				sub(/.*\(/, "", a)
 				gsub(/\W/, "", a)
-- 
2.26.2

